components/tabView/infoPanel.component.tsx

```diff
@@ -70,17 +70,17 @@ const InfoPanel: React.FC<Props> = ({
               </div>
             </div>
             {
-              artist?.user.nominatedBy && (
+              artist?.user.application?.invitation.inviter && (
                 <div id="artistReference" className={'flex justify-end relative w-full h-20 mb-20'}>
                   <img src="/dashedEllipses2.svg" className="absolute" />
                   <div className="absolute uppercase mt-2 mr-8">
                   Referred By:
                     <br />
                     <a
                       className="underline glow-highlight"
-                      href={'/artists/'+artist?.user?.nominatedBy?.artist?.handle}
+                      href={'/artists/'+artist?.user.application?.invitation?.inviter?.artist?.handle}
```

Comment:
Should we link to the User/Inviter? What if the artist was invited by a curator or venue? Wonder if we should display the User link here instead since all profile types (artist, curator, venue) are linked to a User?